go.mod

```diff
@@ -22,6 +22,8 @@ require (
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 )
 
+require github.com/Masterminds/semver v1.5.0
```

Comment:
We have a better library.
https://github.com/aquasecurity/go-version

Reply:
Fixed at https://github.com/aquasecurity/trivy-db/pull/352/commits/603ed00e23a9b5997ae6c02006495a716c962297
That said, I see a problem with `github.com/aquasecurity/go-version/semver` vs `github.com/Masterminds/semver`. Given an incomplete version such as `0.1` (it should be `0.1.0`), `semver.Parse` returns an error (aquasecurity) while `semver.NewVersion("0.1")` (Masterminds) doesn't and fills the version.
See https://go.dev/play/p/YrZDE3Gyypz